app/javascript/packs/referrals/ReferralPanel.tsx

```diff
@@ -7,34 +7,50 @@ import en, { flattenMessages } from "../../locale/en";
 import ja from "../../locale/ja";
 
 import { LoaderIcon } from "brave-ui/components/icons";
-import Arrow from "./Arrow";
 import Information from "./Information";
 
 import routes from "../../views/routes";
 
+import ArrowPointer from "./referralPanel/ArrowPointer";
+import Groups from "./referralPanel/Groups";
+import Stats from "./referralPanel/Stats";
+
+export enum ReferralType {
+  Finalized = "finalized",
```

Comment:
I believe this should be all caps. E.g. https://google.github.io/styleguide/jsguide.html#features-objects-enums

Reply:
https://github.com/brave-intl/publishers/pull/2428/commits/733cfdf408e81200e312f8bbc9f4f6063bc4027f